valhalla/thor/edgestatus.h

```diff
@@ -16,9 +16,11 @@ enum class EdgeSet : uint8_t {
                          // reset due to encountering a complex restriction:
                          // https://github.com/valhalla/valhalla/issues/2103
   kPermanent = 1,        // Permanent - shortest path to this edge has been found
-  kTemporary = 2         // Temporary - edge has been encountered but there could
+  kTemporary = 2,        // Temporary - edge has been encountered but there could
                          //   still be a shorter path to this edge. This edge will
                          //   be "adjacent" to an edge that is permanently labeled.
+  kSkipped = 3           // Skipped - edge has been encountered but was thrown out
```

Comment:
new edge status for shortcuts that were encountered but were not used

Reply:
need to add this to the expansion callback for visualization haha

Reply:
ok

Reply:
so, need help here

Reply:
i mentioned it before i saw how it was used, no worries just ignore my comment :smile:

Reply:
ok)